src/js/view/view.js

```diff
@@ -698,6 +698,18 @@ define([
 
             // pass width, height from jwResize if present
             _resizeMedia(width, height);
+
+            if (_this.model.get('stretching') === 'uniform' && _preview) {
+                var playerAspectRatio = _playerElement.clientWidth / _playerElement.clientHeight;
+                // snap image to edges when the difference in aspect ratio is less than 9%
+                _preview.aspectRatioPromise.then(function(imageAspectRatio) {
+                    if (Math.abs(playerAspectRatio - imageAspectRatio) < 0.09) {
+                        utils.css.style(_preview.el, {
+                            backgroundSize: 'cover'
+                        });
```

Comment:
You need to add or remove the style. This only adds it. Look at the actions mixin again.